warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -175,10 +185,22 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
                 browser_details = selenium_Utils.\
                     get_browser_details(browser, self.datafile, **arguments)
             if browser_details is not None:
+                try:
+                    from pyvirtualdisplay import Display
+                    display = Display(visible=0, size=(1024, 768))
+                    display.start()
+                    pNote("Running in headless mode")
```

Comment:
Hi @Annam16,

Is there a way for the user to specify if they want to run the testcase in headless mode? We can add an argument to the browser_launch keyword called headless_mode, which by default be set to False. The user can then set it True to run the testcases in headless mode.

The current implementation defaults all runs to headless mode and there seems to be no way around that barring uninstalling the headless mode libraries.

Let me know what you think.